packages/react/src/components/stepper/Stepper.module.scss

```diff
@@ -0,0 +1,230 @@
+  cursor: not-allowed;
+}
+
+.step.selected {
+  cursor: default;
+}
+
+.disabled .label {
+  text-decoration: none;
+}
+
+.circle {
+  border-radius: 16px;
+  border: 2px solid var(--hds-stepper-color);
+  text-align: center;
+  width: 28px;
+  height: 28px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-color: var(--hds-stepper-background-color);
+  outline: 2px solid var(--hds-stepper-background-color);
+}
+
+.disabled .circle {
+  border: 2px solid var(--hds-stepper-disabled-color);
+}
+
+.selected .circle {
+  border: 4px solid var(--hds-stepper-color);
+  outline: none;
+  border-radius: 18px;
+}
+
+.number {
+  font-weight: 500;
+  font-size: 16px;
+}
+
+.label {
+  font-weight: 500;
+  font-size: 16px;
+  text-decoration: underline;
+  line-height: 1.5;
+  margin-top: 6px;
+  margin-bottom: 0;
+  padding-left: 2px;
+  padding-right: 2px;
+}
+
+.selected .label {
+  color: var(--color-black-90);
+  text-decoration: none;
+  font-weight: bold;
+}
+
+.filledContainer {
+  background-color: var(--hds-stepper-color);
+  padding: 4px;
+  border-radius: 16px;
+  outline: 2px solid var(--hds-stepper-background-color);
+
+  .filledIcon {
+    color: var(--hds-stepper-background-color);
+    vertical-align: middle;
+  }
+}
+
+.circleContainer {
+  z-index: 2;
+  width: var(--circle-container-size);
+  height: var(--circle-container-size);
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.line {
+  display: inline;
+  position: absolute;
+  line-height: var(--lineheight-l);
+  left: calc(var(--hds-step-width) / 2.0);
+  height: var(--progress-line-height);
+  z-index: 1;
+  transform: translateY(calc((var(--circle-container-size) / 2) + (var(--progress-line-height) / 2)));
+
+  .disabledLine {
+    position: relative;
+    top: -15px;
+    display: inline-block;
+    height: 100%;
+    background-color: var(--hds-stepper-disabled-color);
+  }
+
+  .enabledLine {
+    position: relative;
+    top: -15px;
+    display: inline-block;
+    height: 100%;
+    background-color: var(--hds-stepper-color);
+  }
+}
+
+.heading {
+  margin-top: 26px;
+  font-size: 36px;
+  line-height: 43px;
+  font-weight: bold;
+  width: fit-content;
+
+  &:focus {
+    outline: 3px solid var(--color-coat-of-arms);
+    border: none;
+  }
+  // margin-left: calc((var(--hds-step-width) / 2) - (var(--circle-container-size) / 2));
+}
+
+.scrollButton {
+  z-index: 10;
+  width: 26px;
+  color: var(--hds-stepper-color);
+  position: absolute;
+  top: 7px;
+  background-color: var(--hds-stepper-background-color);
+
+  button {
+    @extend %buttonReset;
+    box-sizing: border-box;
+    width: 100%;
+    height: 26px;
+    overflow: hidden;
+    border-radius: unset;
+    border: 2px solid transparent;
+    display: inline-flex;
+    align-items: center;
+    justify-content: center;
+
+    &:focus {
+      outline: none;
+      border: 2px solid var(--color-coat-of-arms);
```

Comment:
I think it could be wise to create a component-specific variable for this. I do not like the idea that we are using the brand colour variable here. Yes, the colour is the same as the brand colour but this solution makes it harder to change it later on I assume.

Maybe create a `--hds-stepper-focus-color` variable and set it to `--color-coat-of-arms`? It would be also handy if this can be customised. This is needed if the stepper is on a dark background for some reason.

Reply:
- [ ] Create --hds-stepper-focus-color